avivator/src/components/Controller/components/CameraOptions.js

```diff
@@ -11,21 +13,37 @@ import {
 } from '../../../state';
 import { useWindowSize } from '../../../utils';
 
+const useStyles = makeStyles(theme =>
+  createStyles({
+    enabled: {
+      marginLeft: '4px'
+    },
+    disabled: {
+      color: theme.palette.text.disabled,
+      marginLeft: '4px'
+    }
+  })
+);
+
 const CameraOptions = () => {
   const { loader } = useChannelSettings();
   const { useFixedAxis, toggleUseFixedAxis } = useImageSettingsStore();
-  const { setViewerState, viewState } = useViewerStore();
+  const { setViewerState, viewState, use3d } = useViewerStore();
   const { height, width } = useWindowSize();
+  const classes = useStyles();
   const toggleFixedAxisButton = (
     <Grid item xs="auto" key="toggle-fixed-axis">
-      <ToggleButton
-        selected={useFixedAxis}
-        onClick={toggleUseFixedAxis}
-        style={{ padding: 0 }}
-        value={useFixedAxis}
-      >
-        Fix Camera Axis
-      </ToggleButton>
+      <Grid container direction="row">
+        <Checkbox
+          onClick={toggleUseFixedAxis}
+          style={{ padding: 0 }}
+          disabled={!use3d}
+          checked={useFixedAxis}
+        />
+        <Typography className={!use3d ? classes.disabled : classes.enabled}>
```

Comment:
There's probably a better way to do this but I could only spend so long fighting Material UI